src/app/modules/shared_models/keypair_item.nim

```diff
@@ -9,6 +9,7 @@ type
     Profile
     SeedImport
     PrivateKeyImport
+    WatchOnly
```

Comment:
@Khushboo-dev-cpp this refers to a keypair type, there is no watch only keypair, that's the point and the reason why I didn't want to add it. It logically doesn't belong to this enum.

Could we change this somehow?